src/screens/Settings/ComplianceCertificates/Compliance.res

```diff
@@ -0,0 +1,79 @@
+module DownloadCertificateTile = {
+  @react.component
+  let make = (~header, ~onClick) => {
+    <div
+      className="flex flex-col bg-white pt-6 pl-6 pr-8 pb-8 justify-between gap-10 border border-jp-gray-border_gray rounded">
+      <div>
+        <p className="text-fs-16 font-semibold m-2"> {header->React.string} </p>
+      </div>
+      <Button
+        text="Download"
+        buttonSize={Medium}
+        buttonType={Primary}
+        rightIcon={FontAwesome("download-api-key")}
+        onClick
+      />
+    </div>
+  }
+}
+
+@react.component
+let make = () => {
+  let showToast = ToastState.useShowToast()
+  let fetchApi = AuthHooks.useApiFetcher()
+
+  let downloadURL = Window.env.dssCertificateUrl->Option.getOr("")
+  let downloadPDF = () => {
+    let currentDate =
+      Date.now()
+      ->Js.Date.fromFloat
+      ->Date.toISOString
+      ->TimeZoneHook.formattedISOString("YYYY-MM-DD HH:mm:ss")
+
+    //? - For localtesting this condn added
+    if downloadURL->LogicUtils.isNonEmptyString {
+      open Promise
+      fetchApi(downloadURL, ~method_=Get, ())
```

Comment:
async await can be used to maintain consistency throughout codebase

Reply:
Cant use async/await because useGetMethod adds an authorization header. Since useGetMethod calls apiFetcher with a header, a new parameter would be needed in useGetMethod to handle cases without the header